packages/terminal/src/widget.ts

```diff
@@ -350,35 +338,17 @@ class Terminal extends Widget {
    * Set the stylesheet.
    */
   private _setStyle(): void {
-    // Set the fg and bg colors of the terminal and cursor.
-    this._sheet.innerHTML = (`
-      #${this.node.id} {
-        background: ${this._background};
-        color: ${this._color};
-      }
-      #${this.node.id} .xterm-viewport, #${this.node.id} .xterm-rows {
-        background-color: ${this._background};
-        color: ${this._color};
-      }
-      #${this.node.id} .terminal.focus .terminal-cursor.blinking {
-          animation: ${this.node.id}-blink-cursor 1.2s infinite step-end;
-      }
-      @keyframes ${this.node.id}-blink-cursor {
-          0% {
-              background-color: ${this._color};
-              color: ${this._background};
-          }
-          50% {
-              background-color: transparent;
-              color: ${this._color};
-          }
-      }
-    `);
+    if (this._theme === 'dark') {
```

Comment:
This can be simplified:

```
this.toggleClass(TERMINAL_LIGHT_THEME, theme === 'light');
this.toggleClass(TERMINAL_DARK_THEME, them === 'dark');
```